integration-tests/smoke/automation_test.go

```diff
@@ -133,6 +134,10 @@ func SetupAutomationBasic(t *testing.T, nodeUpgrade bool) {
 				return
 			}
 
+			// Use the name to determine if this is a log trigger or not
+			// Not an elegant solution but it reuses the most code and avoids a lot of duplication
+			isLogTrigger := name == "registry_2_1_logtrigger"
```

Comment:
seems fine to me! I would maybe get rid of the comment though. Feels like it doesn't add much.

Reply:
haha, it is so hacky that i want to give my reasoning :) I will simplify it.